app/src/main/res/layout-land/administrator_controls_account_actions_view.xml

```diff
@@ -18,14 +18,13 @@
       android:layout_width="match_parent"
       android:layout_height="wrap_content"
       android:background="@color/mid_grey_30"
-      android:fontFamily="sans-serif-medium"
+      style="@style/Subtitle2"
```

Comment:
Please revert all changes in this file because this PR is for #1979 which focuses only on `administrator_controls_app_information_view.xml` and not on `administrator_controls_account_actions_view.xml`